homework3/script1.js

```diff
@@ -0,0 +1,27 @@
+const lettersDigitsCounter = (string) => {
+  let letterCount = 0;
+  let digitCount = 0;
+  let symbolCount = 0;
+
+  for (let i = 0; i < string.length; i++) {
+    const char = string[i];
+    switch (true) {
+      case /[a-zA-Z]/.test(char):
+        letterCount++;
+        break;
+      case /[0-9]/.test(char):
+        digitCount++;
+        break;
+      default:
+        symbolCount++;
+        break;
+    }
+  }
+  return {
+    letters: letterCount,
```

Comment:
you can rename your variable like this
`let letters = 0;
  let digits = 0;
  let symbols = 0;`
so at the end you can return this
`return {
letters,
digits,
symbols
}`